.github/workflows/continuous-integration.yml

```diff
@@ -316,24 +318,76 @@ jobs:
   e2e-tests:
     name: "Tests: End-to-end"
     runs-on: ubuntu-latest
+    env:
+      CI: 1
```

Comment:
Sollte eigentlich gesetzt sein https://docs.github.com/en/enterprise-cloud@latest/actions/learn-github-actions/environment-variables#default-environment-variables

Aber eventuell ist das true nicht so einfach zum detektieren

Reply:
Ah, von dort kommt das her. Ich glaubs das Problem ist, dass `-eq` für integer-Comparison gedacht ist. In shell gibt es keine Boolean-Werte, sprich `true` ist ein String.

I'll give it another try to fix it.

Reply:
Am besten einen String draus machen. Habe ich in einem Stackoverflow Post gefunden.

Reply:
Fixed with https://github.com/ecamp/ecamp3/pull/2575/commits/a01f0d2090f348ac5f57b3632aa3a71aa9306422